2-alcance/README.md

```diff
@@ -1,3 +1,187 @@
+* Un lugar físico de contacto (oficina)
+* Opción de consumir contenido únicamente estando registrado
+
+### 4. DETALLES
+#### 4.1. Requisitos funcionales de usuarios
+Como se ha comentado en el anterior apartado, la página web dispondrá inicialmente de 3 perfiles de usuario distinto (usuario administrador, usuario voluntario y usuario no registrado), cada uno con requisitos funcinales específicos.
+
+Así mismo, existirán 3 niveles jerarquicos: el primer nivel está formado por el usuario no registrado, el segundo nivel por el usuario voluntario y finalmente el tercer nivel por el usuario administrador del sitio web. Los usuarios de cada nivel tendrán asignadas sus funcionalidades pero también heredarán las de los niveles inferiores (el usuario administrador podrá acceder a las funcionalidades del voluntario y del no registrado; y el usuario voluntario podrá acceder a las del usuario no registrado). En la siguiente imagen se muestra la relación entre niveles. 
+
+![Niveles de usuarios](/2-alcance/Niveles.png)
+
+A continuación se presentan los requisitos de funcionamiento especificos de cada perfil de usuario.
+
+#### 4.1.1. Usuario no registrado.
+El usuario no registrado se describe como el visitante ocasional y dispondrá de los siguientes requisitos funcionales: 
+
+  * **Acceso al contenido público:** este usuario tendrá acceso a todo el contenido que esté oficialmente publicado en el sitio web. No tendrá que registrarse para poder consumir el contenido. Tendrá acceso a: noticias, reportajes, articulos, entrevistas, historias reales, consejos, quiénes somos y contacto.
+
+  * **Registrarse:** tendrán la opción disponible para registrarse en *hoyportimañanapormi.com* opción para convertirse en un usuario voluntario. 
+
+
+#### 4.1.2. Usuario voluntario (registrado)
+El usuario registrado heredará las funcionalidades del usuario no registrado y dispondrá de nuevos requisitos funcionales:
+
+  * **Registrarse:** deberan rellenar un formulario con los datos necesario y aceptar los términos legales sobre difusión y gestión de datos. 
+
+  * **Autentificarse:** para poder acceder al contenido los usuarios voluntarios deberan hacer un log-in (usuario y contraseña).
+
+  * **Realizar comentarios de mejoras y cambios:** la página web tendrá una funcionalidad para que los usuarios puedan realizar comentarios (privados) de mejoras o cambios en el contenido público. 
+
+  * **Compartir información:** los usuarios voluntarios podrán compartir contenido relacionado con la igualdad de género. Para ello, tendrán una lugar específico donde poder adjuntar la información. 
+
+
+#### 4.1.3. Usuario administrador
+El usuario administrador heredará las funcionalidades disponibles para los iveles inferiroes de usuario de la página web, y además dispondrá de los siguinetes requisitos funcionales:
+
+- **Gestor de contenido:**
+  * **Agregar contenido**
+  * **Editar contenido**
+  * **Eliminar contenido**
+
+
+- **Gestor de los perfiles registrados:**
```

Comment:
Estas tareas, entiendo que por más complejas, son las que menos definidas tenéis. Deberíais intentar concretarlas aún más, tanto qué tareas específicas estarán a manos del administrador, como en qué consistirán exactamente esas tareas.